ivy/functional/ivy/elementwise.py

```diff
@@ -5092,6 +5092,223 @@ def subtract(
     ret
         an array containing the element-wise differences.
 
+         Functional Examples
+    ------------------
+
+    With: code:`ivy.Array` inputs:
+
+    >>> x1 = ivy.array([1, 0, 1, 1])
+    >>> x2 = ivy.array([1, 0, 0, -1])
+    >>> y  =  ivy.subtract(x1, x2)
+    >>> print(y)
+    ivy.array([0, 0, 1, 2])
+
+    >>> x1 = ivy.array([1, 0, 1, 0])
+    >>> x2 = ivy.array([0, 1, 0, 1])
+    >>> y = ivy.subtract(x1, x2)
+    >>> print(y)
+    ivy.array([ 1, -1,  1, -1])
+
+    >>> x1 = ivy.array([1, -1, 1, -1])
+    >>> x2 = ivy.array([0, -1, 1, 0])
+    >>> y = ivy.zeros(4)
+    >>> ivy.subtract(x1, x2, out=y)
+    >>> print(y)
+    ivy.array([ 1,  0,  0, -1])
+
+    >>> x1 = ivy.array([1, -1, 1, -1])
+    >>> x2 = ivy.array([0, -1, 1, 0])
+    >>> y = ivy.subtract(x1, x2, out=x1)
+    >>> print(y)
+    vy.array([ 1,  0,  0, -1])
+
+    With a mix of :code:`ivy.Array` and :code:`ivy.NativeArray` inputs:
+
+    >>> x1 = ivy.native_array([1, 2])
+    >>> x2 = ivy.array([1, 2])
+    >>> y = ivy.subtract(x1, x2)
+    >>> print(y)
+    ivy.array([0, 0])
+
+    >>> x1 = ivy.native_array([1, -1])
+    >>> x2 = ivy.array([0, 1])
+    >>> y = ivy.subtract(x1, x2)
+    >>> print(y)
+    ivy.array([ 1, -2])
+
+    >>> x1 = ivy.native_array([1, -1, 1, -1])
+    >>> x2 = ivy.native_array([0, -1, 1, 0])
+    >>> y = ivy.zeros(4)
+    >>> ivy.subtract(x1, x2, out=y)
+    >>> print(y)
+    ivy.array([ 1,  0,  0, -1])
+
+    >>> x1 = ivy.native_array([1, 2, 3, 4])
+    >>> x2 = ivy.native_array([0, 2, 3, 4])
+    >>> y = ivy.zeros(4)
+    >>> ivy.subtract(x1, x2, out=y)
+    >>> print(y)
+    ivy.array([1, 0, 0, 0])
```

Comment:
This is also the same as the one above.

Reply:
This happens many times, we do not want to have multiple examples that say the same thing since that just takes up space. If you have a new example, make sure its different from the one before more than just values different.